source/Calamari.Aws/AwsExtension.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using Autofac;
+using Calamari.Aws.Commands;
+
+namespace Calamari.Aws
+{
+    public class AwsExtension : ICalamariExtension
+    {
+        readonly Dictionary<string, Type> commandTypes = new Dictionary<string, Type>
+        {
+            {"apply-aws-cloudformation-changeset", typeof(ApplyCloudFormationChangesetCommand)},
```

Comment:
I'm not convinced this is better than the attribute. It separates the name of the command from the command itself.